plugin.json

```diff
@@ -6,13 +6,13 @@
     "posthogVersion": ">= 1.24.0",
     "config": [
         {
-          "key": "percentage",
-          "hint": "Reduces events flowing in to above %",
-          "name": "% of events to keep",
-          "type": "string",
-          "order": 1,
-          "default": "100",
-          "required": false
+            "key": "percentage",
+            "hint": "Reduces events flowing in to above %",
```

Comment:
"above %" you mean "more than this percentage" or "the percentage in the input field above"? I had to do a double take to understand, so perhaps it might make sense to revisit the wording.